doc/references.rst

```diff
@@ -1,56 +1,6 @@
 References
 ==========
 
-.. [Ahrens2012]
-    Ahrens, J.,
-    *Analytic Methods of Sound Field Synthesis*.
-    (Springer, Berlin Heidelberg, 2012),
-    doi:`10.1007/978-3-642-25743-8 <https://doi.org/10.1007/978-3-642-25743-8>`__
-
-.. [Moser2012]
-    Möser, M.,
-    *Technische Akustik*.
-    (Springer, Berlin Heidelberg, 2012),
-    doi:`10.1007/978-3-642-30933-5 <https://doi.org/10.1007/978-3-642-30933-5>`__
-
-.. [SporsAhrens2010]
-    Spors, S., Ahrens, J.,
-    “Analysis and Improvement of Pre-equalization in 2.5-dimensional Wave Field Synthesis,”
-    in *128th Conv. Audio Eng. Soc.*, London, UK (2010), Paper 8121,
-    `pdf <https://www.int.uni-rostock.de/fileadmin/user_upload/publications/spors/2010/AES128_Spors_etal_WFS_preeq.pdf>`__
-
-.. [SporsAhrens2009]
-    Spors, S., Ahrens, J.,
-    “Spatial Sampling Artifacts of Wave Field Synthesis for the Reproduction of Virtual Point Sources,”
-    in *126th Conv. Audio Eng. Soc.*, Munich, Germany (2009), Paper 7744,
-    `pdf <https://www.int.uni-rostock.de/fileadmin/user_upload/publications/spors/2009/AES126_Spors_etal_WFS_spatial_aliasing.pdf>`__
-
-.. [Spors2016]
-    Spors, S., Schultz, F., Rettberg, T.,
-    “Improved Driving Functions for Rectangular Loudspeaker Arrays Driven by Sound Field Synthesis,”
-    in *42nd Ger. Ann. Conf. Acoust. (DAGA)*, Aachen, Germany (2016),
-    `pdf <https://www.int.uni-rostock.de/fileadmin/user_upload/publications/spors/2016/Spors_etal_DAGA_SFS_rect_array.pdf>`__
-
-.. [Spors2008]
-    Spors, S., Rabenstein, R., and Ahrens, J.,
-    “The Theory of Wave Field Synthesis Revisited,”
-    in *124th Conv. Audio Eng. Soc.*, Amsterdam, Netherlands (2008), Paper 7358,
-    `pdf <https://www.int.uni-rostock.de/fileadmin/user_upload/publications/spors/2008/AES124_Spors_WFS_Theory.pdf>`__
-
-.. [Wierstorf2014]
-    Wierstorf, H.,
-    *Perceptual Assessment of Sound Field Synthesis*,
-    Ph.D. dissertation, Technische Universität Berlin, Berlin, Germany, 2014,
-    doi:`10.14279/depositonce-4310 <http://dx.doi.org/10.14279/depositonce-4310>`__
-
-.. [AllenBerkley1979]
-    Allen, J. B., Berkley, D. A.,
-    “Image method for efficiently simulating small‐room acoustics,”
-    J. Acoust. Soc. Am. **65**, 943-950 (1979),
-    doi:`10.1121/1.382599 <https://doi.org/10.1121/1.382599>`__
-
-.. [Borish1984]
-    Borish, J.,
-    “Extension of the image model to arbitrary polyhedra,”
-    J. Acoust. Soc. Am. **75**, 1827-1836 (1984),
-    doi:`10.1121/1.390983 <https://doi.org/10.1121/1.390983>`__
+.. bibliography:: references.bib
+    :style: plain
```

Comment:
Are numbered references really the best choice here?
What about using the default `alpha` style?

Reply:
At the moment there are only two choices with sphinxcontrib-bibtex, one is the numbered version and the other one is `alpha`. The latter one I dislike very much as it is neither the proper names of the authors nor an ordered list of references, but it's also fine with me to switch to that one.
if we stick with `plain` we have to remember that we have to use [a], [b], ... for footnotes instead of [1], [2], ....

Reply:
Of course it would be nicest to have full author names and years, but I think that's unrealistic right now.

I guess we'll have to choose the lesser of two evils.
If you prefer the numbered style, I'm fine with it.

Reply:
I'm still not completely decided as I also see some advantages of `alpha`.
I will do a few more tests and then propose what I would prefer.

Reply:
I have switched now to `alpha`. It is indeed not perfect, but you can still use the abbreviation to some amount like a name. I tested this also in the theory repository.

If we are motivated we could at a later stage have a look if we would like to add another option to sphinxcontrib-bibtex itself.

Reply:
OK, cool. And yes, it would be great if this would be added to `sphinxcontrib-bibtex`, but I'm not going to work on that ...